src/price_data.py

```diff
@@ -133,6 +133,91 @@ def _get_price_binance(
         average_price = total_cost / total_quantity
         return average_price
 
+    @misc.delayed
+    def _get_price_coinbase(self, base_asset: str, utc_time: datetime.datetime, quote_asset: str, swapped_symbols: bool = False) -> decimal.Decimal:
+        # Use Coinbase Pro prices
+        return self._get_price_coinbase_pro(base_asset, utc_time, quote_asset, swapped_symbols)
+
+    @misc.delayed
+    def _get_price_coinbase_pro(self, base_asset: str, utc_time: datetime.datetime, quote_asset: str, swapped_symbols: bool = False) -> decimal.Decimal:
+        """Retrieve price from Coinbase Pro official REST API.
+
+        The price is calculated as the average price of the opening and closing price
+        around 1 minute around `utc_time`.
+
+        None existing pairs like `TWTEUR` are calculated as
+        `TWTBTC * BTCEUR`.
+
+        Documentation: https://docs.pro.coinbase.com
+
+        Args:
```

Comment:
This tag is only for function parameters and I wouldn't place the API description in the function docstring. Just place it in the code.